docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -22,29 +22,50 @@ for more information on these two file transfer sources.
-    env variable) to setup volumes with Docker, which on a Mac can default
-    to a subdirectory of /var. In order to get ``cwltool`` working on your
-    Mac, you need to set your TMPDIR to be under one of the shared volumes
-    in Docker for Mac. You can do this by doing something similar to the
-    following:
-    ::
+.. note:: Our tutorials all assume you are using bash on an Ubuntu machine. Beginning with Mac OS 10.15 Catalina, the default shell has changed from bash to zsh. In almost all cases (especially in our tutorials), syntax between bash-on-Ubuntu, bash-on-Mac, and zsh-on-Mac is the same, but it is possible for slight differences to arise.
 
-        export TMPDIR=/tmp/docker_tmp
+See `Docker for Mac <https://docs.docker.com/engine/installation/mac/>`__ for information on installing Docker for Mac. With the exception of some installation and possible folder permission differences (see next FAQ below), usage of the Dockstore CLI on a Mac is more or less the same as it is on Ubuntu.
 
 By default, Docker for Mac allocates fewer resources (CPU, Memory, Swap)
 to containers compared to what is available on your host machine. You
 can change what it allocates using the Docker for Mac GUI under
 ``Preferences > Advanced`` as described
 `here <https://docs.docker.com/docker-for-mac/#advanced>`__.
 
-* The default allocation can cause workflows or tools to fail without informing the user with a memory or resource related error message. If you find that your workflow or tool is behaving differently on a Mac compared to a similarly resourced Ubuntu environment, you can try increasing the resources allocated to Docker on the Mac to resolve the discrepancy.
+The default allocation can cause workflows or tools to fail without informing the user with a memory or resource related error message. If you find that your workflow or tool is behaving differently on a Mac compared to a similarly resourced Ubuntu environment, you can try increasing the resources allocated to Docker on the Mac to resolve the discrepancy. If you are using WDL, see also our notes on :doc:`local Cromwell configuration files`</advanced-topics/dockstore-cli/local-cromwell-config>`.
+
+Why am I getting "Mount denied" errors when launching workflows on Mac?
+-----------------------------------------------------------------------
+Docker behaves a bit differently on a `Mac <https://docs.docker.com/docker-for-mac/osxfs/#/namespaces>`__ than on a typical Ubuntu machine. On a Mac, by default, the only shared volumes Docker can access are /Users, /Volumes, /tmp, and /private. This is problematic because many workflow executors, such as `cwltool`, uses your TMPDIR (the :ref:`dict environment variable`) to setup volumes with Docker. Since `$TMDIR` on a Mac can default to a subdirectory of /var, which Docker usually cannot access, you may run into an error. When using cwltool, the error will look something like this:
```

Comment:
"to setup volumes with Docker" -> "to set up volumes with Docker" -- I had to look it up, but setup is one word when used as a noun, but two words when "set" is a verb.

Reply:
Good catch!